src/plugins/creditor/declaration.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import deepFreeze from "deep-freeze";
+import type {PluginDeclaration} from "../../analysis/pluginDeclaration";
+import type {NodeType, EdgeType} from "../../analysis/types";
+import {NodeAddress, EdgeAddress} from "../../core/graph";
+
+export const nodePrefix = NodeAddress.fromParts(["sourcecred", "creditor"]);
+export const edgePrefix = EdgeAddress.fromParts(["sourcecred", "creditor"]);
+
+export const CreditorEntryType: NodeType = deepFreeze({
+  name: "CredNode",
+  pluralName: "CredNodes",
+  prefix: NodeAddress.append(nodePrefix, "credNode"),
+  defaultWeight: 0,
+  description:
+    "An creditor node, describing a scoped improvement to a project from proposal to completion. \
+    Creditor nodes can be nested  arbitrarily (but non-circularly) to represent complet projects and scopes",
```

Comment:
Suggest "A node manually created by the Creditor". "Scoped improvement..." imo confuses the matter, esp, when we use Creditor Nodes for stuff like meetings. Talking about how they can be nested is just gonna confuse people.

Note: This text shows up in the UI (in the weights pane atm)